easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.9-GCCcore-10.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'bsddb3'
+version = '6.2.9'
+
+homepage = 'https://pypi.org/project/bsddb3/'
+description = """bsddb3 is a nearly complete Python binding of the
+Oracle/Sleepycat C API for the Database Environment, Database, Cursor,
+Log Cursor, Sequence and Transaction objects."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['70d05ec8dc568f42e70fc919a442e0daadc2a905a1cfb7ca77f549d49d6e7801']
+
+osdependencies = [('libdb-dev', 'libdb-devel')]
+
+builddependencies = [('binutils', '2.35')]
+dependencies = [('Python', '3.8.6')]
+
+use_pip = True
+download_dep_fail = True
+
```

```suggestion
sanity_pip_check = True 

```

And `sanity_pip_check` should be set.